en/docs/get-started/quick-start-guide.md

```diff
@@ -91,25 +95,41 @@ Follow this procedure to further develop the API by adding a new resource and si
 
```

Comment:
We might need to update steps 3 and 4 to align with the new UIs. Might have to update the screenshot related to step 4 too.

Reply:
Done